README.md

```diff
@@ -65,14 +65,15 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [turkanime.net](https://turkanime.net/) - Contains Turkish Subs
+* [tranimeizle.net](https://tranimeizle.net) - Contains Turkish Subs
```

Comment:
this doesn't seem to load for me, does it work for you?

Reply:
both of them are working